backend/user-service/psql.js

```diff
@@ -0,0 +1,39 @@
+require('dotenv').config();
+const { Pool } = require('pg');
+
+const pool = new Pool({
+    host     : process.env.PSQL_HOSTNAME,
+    database : process.env.PSQL_database,
```

Comment:
Just naming inconsistent thing, can consider rename the PSQL_database var to all CAPS.

Reply:
Resolved